go/cmd/dolt/dolt.go

```
@@ -144,32 +151,56 @@ func runMain() int {
 
-		port, err := strconv.ParseUint(*portStr, 10, 16)
+	// if disabled, err := strconv.ParseBool(*metricsDisabled); err == nil && !disabled {
+	// 	host := dEnv.Config.GetStringOrDefault(env.MetricsHost, env.DefaultMetricsHost)
+	// 	portStr := dEnv.Config.GetStringOrDefault(env.MetricsPort, env.DefaultMetricsPort)
+	// 	insecureStr := dEnv.Config.GetStringOrDefault(env.MetricsInsecure, "false")
 
-		if err != nil {
-			log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid port.", env.MetricsPort, *portStr))
-			return emitter
-		}
+	// 	port, err := strconv.ParseUint(*portStr, 10, 16)
 
-		insecure, err := strconv.ParseBool(*insecureStr)
+	// 	if err != nil {
+	// 		log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid port.", env.MetricsPort, *portStr))
+	// 		return emitter
+	// 	}
 
-		if err != nil {
-			log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid true/false value", env.MetricsInsecure, *insecureStr))
-		}
+	// 	insecure, err := strconv.ParseBool(*insecureStr)
 
-		hostAndPort := fmt.Sprintf("%s:%d", *host, port)
-		conn, _ := dEnv.GrpcConnWithCreds(hostAndPort, insecure, nil)
+	// 	if err != nil {
+	// 		log.Println(color.YellowString("The config value of '%s' is '%s' which is not a valid true/false value", env.MetricsInsecure, *insecureStr))
+	// 	}
 
-		return events.NewGrpcEmitter(conn)
-	}
+	// 	hostAndPort := fmt.Sprintf("%s:%d", *host, port)
+	// 	conn, _ := dEnv.GrpcConnWithCreds(hostAndPort, insecure, nil)
+
+	// 	return events.NewGrpcEmitter(conn)
+	// }
 
 	return emitter
 }
+
+func processEventsDir(args []string, dEnv *env.DoltEnv) error {
+	if len(args) > 0 {
+		if args[0] != commands.SendMetricsCommand {
+			disabled, err := events.AreMetricsDisabled(dEnv)
+			if !disabled && err == nil {
+				cmd := exec.Command("dolt", commands.SendMetricsCommand)
+
+				if err := cmd.Start(); err != nil {
+					log.Print(err)
```

I think we should fail silently until a later date when we support a verbose logging mode.